### PR TITLE
Delete Merging View implementation

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

Maybe issues are related to the merging view (see issues). After the tie lines refactoring, the sub-networks modeling will be the only tool box for CGMES profiles export.

**What kind of change does this PR introduce?**

The merging view as IIDM implementation is no more available, too risky to use too hard and costly to fix directly.

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [X] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

**Comment by mathbagu** [comment]
Don't forget to remove references to the merging view in the website. A quick search gives me the list of artifacts, but there are potential other pages.

File: cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/CgmesConformity3ConversionTest.java [hunk 1]
```diff
@@ -114,22 +113,19 @@ void microGridBaseCaseBEMergingViewNL() {
         // Both networks have the same number of dangling lines
         assertEquals(nDlBE, nDlNL);
 
-        Network n = MergingView.create("be-nl", "CGMES");
-        n.merge(nl, be);
+        be.merge(nl);
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/CgmesConformity3ConversionTest.java:116]: We see here that we have an equivalence now between the merging view and the destructive merge for equipments.

File: cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/modified/CgmesConformity1ModifiedConversionTest.java [hunk 2]
```diff
@@ -510,65 +509,6 @@ void microBEReactivePowerGen() {
...
-    @Test
-    void microEquivalentBranchAtBoundaryCompareMerges() {
-        final String tieLineId = "78736387-5f60-4832-b3fe-d50daf81b0a6 + 7f43f508-2496-4b64-9146-0a40406cbe49";
-        ReadOnlyDataSource assembled = CgmesConformity1ModifiedCatalog.microGridBaseCaseAssembledEquivalentBranchAtBoundary().dataSource();
-        ReadOnlyDataSource ds1 = CgmesConformity1ModifiedCatalog.microGridBEEquivalentBranchAtBoundary().dataSource();
-        ReadOnlyDataSource ds2 = CgmesConformity1Catalog.microGridBaseCaseNL().dataSource();
-        compareMerges(tieLineId, assembled, ds1, ds2);
-    }
-
-    private static void compareMerges(String tieLineId, ReadOnlyDataSource dsAssembled, ReadOnlyDataSource ds1, ReadOnlyDataSource ds2) {
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/modified/CgmesConformity1ModifiedConversionTest.java:540]: Tie lines in the merging view and in the destructive merge are now equivalent. These tests can be deleted.

File: cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/CgmesConformity3ConversionTest.java (hunk 1 above)
Comment (miovd) [reply cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/CgmesConformity3ConversionTest.java:116]: I think this test can be merged with the test above `microGridBaseCaseBEMergedWithNL`, they check the same thing now...

**Review by miovd — COMMENTED** [review]
Mostly okay for me, one small comment.
Before deleting the merging view, be sure to check that the EMF process is equivalent with sub-networks and the merging view :)

**Review by geofjamg — APPROVED** [review]

**Comment by olperr1** [comment]
> Don't forget to remove references to the merging view in the website. A quick search gives me the list of artifacts, but there are potential other pages.

The merging view is also used in the "powsybl/powsybl-tutorials" repository.

**Pull request merged**